plugins/scaffolder-backend/src/scaffolder/stages/templater/cookiecutter.ts

```diff
@@ -21,7 +21,7 @@ import path from 'path';
 import { runCommand } from './helpers';
 import { TemplaterBase, TemplaterRunOptions } from './types';
 
-const commandExists = require('command-exists-promise');
+const commandExists = require('command-exists');
```

Comment:
Doesn't `command-exists` provide typings that you could use here?

Reply:
See https://www.npmjs.com/package/@types/command-exists

Reply:
Updated it 👍